var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -170,8 +172,20 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
 
     # External package options
     variant("dtk", default=False, description="Enable DataTransferKit (deprecated)")
-    variant("scorec", default=False, description="Enable SCOREC")
     variant("mesquite", default=False, description="Enable Mesquite (deprecated)")
+    variant("pytrilinos2", default=False, description="Enable PyTrilinos2)")
+    variant("scorec", default=False, description="Enable SCOREC")
+
+    resource(
```

Comment:
I understand why this is here short-term, but obviously long-term pytrilinos2 will live within Trilinos.  Not sure if we want to add it like this now because it will require Kim to grant public pull access to that branch.